src/hid-sensor.cpp

```diff
@@ -339,6 +348,34 @@ uint32_t hid_sensor::fps_to_sampling_frequency( rs2_stream stream, uint32_t fps
     else
         return fps;
 }
+void hid_sensor::set_imu_sensitivity( rs2_stream stream, float value ) 
+{
+    _imu_sensitivity_per_rs2_stream[stream] = value;
+}
+
+float hid_sensor::get_imu_sensitivity_raw_values( rs2_stream stream )
+{
+    if( _imu_sensitivity_per_rs2_stream.find( stream ) != _imu_sensitivity_per_rs2_stream.end() )
+        return _imu_sensitivity_per_rs2_stream[stream];
+     else 
+        // gyro sensitivity default value is +-1000 therefore returning 30.5 resolution
```

Comment:
Can we align the comment Ohad added elsewhere about default is +-2000?
Please sync with him and update him comment as well

Reply:
Done